component/widget.js

```diff
@@ -54,9 +64,9 @@ export default class Widget extends React.Component {
       <div className="item">
         <h3 className="tagline">Should I Deploy Today?</h3>
         <h2 id="text">{this.state.reason}</h2>
-        <button type="button" id="reload" onClick={this.onClick}>
-          Hit me again
-        </button>
+        <span id="reload">
+          Hit <span className="space-btn">Space</span> to get another reason
```

Comment:
Make this actionable as well for mobile

Reply:
I tried to attach a click event
but the problem is, after clicking the element, it stays focused
after that, when pressing the spacebar, the event is fired twice, for click (because it is focused now), and the keydown events
so what I am thinking is, we can use media queries to hide the element with click event listener, on desktops and show only the element with keydown event, and the opposite for the other screens
what do you say?

Reply:
I think I figured it out, I replace ```<span id="reload">``` with ```<button id="reload">``` that is why it is focused when clicking
reverting to ```<span id="reload">``` seem to have fixed the problem

Reply:
Looking good!